### PR TITLE
DM-23186: add pre-commit hook

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

**Comment by r-owen** [comment]
Might you suggest a better wording for the README? I want to use the same boilerplate in all my packages.

File: README.rst [hunk 1]
```diff
@@ -2,7 +2,7 @@ Object-oriented interface to `Service Abstraction Layer <https://docushare.lsstc
 
 `Documentation <https://ts-salobj.lsst.io>`_
 
-The package is compatible with setuptools, as well as the Vera Rubin LSST DM's ``eups`` package management system and ``scons`` build system.
+The package is compatible with setuptools, as well as the Vera C. Rubin LSST DM's ``eups`` package management system and ``scons`` build system.
```
Comment (timj) [inline README.rst]: How about:
```markdown
The package is compatible with setuptools, as well as the [``eups`` package management system](https://github.com/RobertLuptonTheGood/eups) and ``scons`` build system.
```
?

Comment (timj) [reply README.rst]: (except you are using rST and not markdown...)

**Comment by r-owen** [comment]
By the way: I thought pretty hard about commit hooks and I'm still a bit uneasy. There doesn't seem to be a foolproof way to automatically enable them for everyone who clones the package. I chose to this solution as being simple and not requiring the pre-commit Python package (which as far as I can see doesn't really gain us much for such a simple hook). If you have any thoughts on this I would welcome them.

**Comment by timj** [comment]
I think it's best to try things out and see what happens. It's going to be interesting to see if people set things up properly. At least you should be able to spot things in the pull requests.

File: README.rst (hunk 1 above)
Comment (r-owen) [reply README.rst]: Sounds good, though I will make only "eups" the link.

**Pull request merged**